src/components/Shared/Filters/FilterListItemMultiselect.tsx

```diff
@@ -122,7 +127,59 @@ export const FilterListItemMultiselect: React.FC<Props> = ({
             />
           </ListItem>
         ))
-      )}
+      ) : filter.filterKey === 'status' ||
+        filter.filterKey === 'contact_status' ? (
+        <ListItem>
+          <Autocomplete
```

Comment:
This is identical to the autocomplete besides the `groupBy` and `options` sorting, right? Ideally, we would extract it into a separate component that could be used in both places, and we pass in optional `groupBy` and `sortOptions` props.